pwm_board_config:
# If your board doesn't handle inputs (sensors) then defining device with a data_type.type:'inputs' will make the
# node throw an error
  data_types:
    'servo180':
      type: 'output'
      range: [-90,90]
      accel: 10
    'servo90':
      type: 'output'
      range: [-45,45]
      accel: 10
    'motor100':
      type: 'output'
      range: [-100,100]
      max: 50.0
      min: -50.0
      deadband: [-2,2]
      deadband_offset: [-20,20]
      accel: 10
    'attopilot':
      type: 'input'

  device:
    'thruster_left':
      'pin': 0
      'type': 'motor100'
    'thruster_right':
      'pin': 1
      'type': 'motor100'
    'battery1_voltage_raw':
      'pin': 8
      'type': 'attopilot'
    'battery1_current_raw':
      'pin': 9
      'type': 'attopilot'
    'battery2_voltage_raw':
      'pin': 10
      'type': 'attopilot'
    'battery2_current_raw':
      'pin': 11
      'type': 'attopilot'
